guides/source/caching_with_rails.md

```diff
@@ -383,10 +552,11 @@ You can access the cache by calling `Rails.cache`.
 
 #### Connection Pool Options
 
-By default, [`:mem_cache_store`](#activesupport-cache-memcachestore) and
-[`:redis_cache_store`](#activesupport-cache-rediscachestore) are configured to use
-connection pooling. This means that if you're using Puma, or another threaded server,
-you can have multiple threads performing queries to the cache store at the same time.
+[`:mem_cache_store`](#activesupport-cache-memcachestore) and
+[`:redis_cache_store`](#activesupport-cache-rediscachestore) are configured to
+use connection pooling. This means that if you're using Puma, or another
+threaded server, you can have multiple threads performing queries to the cache
+store at the same time.
```

Comment:
Solid Cache uses connection pooling as well, but that is configured in the `database.yml`, so I think we are good not mentioning it here.